core/src/main/java/org/mule/processor/AsyncInterceptingMessageProcessor.java

```diff
@@ -72,7 +72,7 @@ public WorkManager getWorkManager() throws MuleException
 
     public void start() throws MuleException
     {
-        if (workManager != null)
+        if (workManager != null && !workManager.isStarted())
```

Comment:
I dont' remember exactly. But a bug was unveiled by this refactor and this was part of the fix.